src/components/Launchpad/Content.tsx

```diff
@@ -26,15 +26,18 @@ const Content: React.FC<{guildpad: GuildpadConfig, rarity?: string }>= ({guildpa
     // const ratio = guildpad.inoDetails.ratio && guildpad.inoDetails.ratio
     // const boxes = guildpad.inoDetails.boxes ?? 'TBA'
     const start = guildpad.date.start ?? 'TBA'
-    const end = guildpad.date.end ?? 'TBA'
     const distribution = guildpad.distribution
     const initMarketCap = 'TBA'
     const initTokenCirc = 'TBA'
+    const inoPrice = guildpad.inoDetails.priceFiat ?? 'TBA'
 
     // Remove last 3 digits on asOfPriceInProjectToken for moment format
-    const newEpoch = (guildpad.asOfPriceInProjectToken - (guildpad.asOfPriceInProjectToken % 1000)) / 1000
+    let newEpoch = (guildpad.asOfPriceInProjectToken - (guildpad.asOfPriceInProjectToken % 1000)) / 1000
```

Comment:
This is not acceptable. If asOfPriceInProjectToken is in milliseconds, use it instead of converting it in unix time 

https://stackoverflow.com/questions/35184003/moment-js-convert-milliseconds-into-date-and-time